microsoft-edge/devtools-guide-chromium/experimental-features.md

```diff
@@ -0,0 +1,69 @@
+---
+description: The latest experimental features in Edge DevTools
+title: Experimental features
+author: hxlnt
+ms.author: raweil
+ms.date: 06/15/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, experiment
+---
+
+# Experimental features
+
+The Edge DevTools periodically offers experimental features that are released while still in development. By enabling an experiment, you are able to test and [provide early feedback](#Providing-feedback-on-experimental-features) on these features before they are released broadly.
+
+Experiments are available in all channels of Edge. Use the Edge Canary channel to get the very latest experiments.
+
+## How to enable experiments
+
+To enable an experiment, open DevTools by pressing `Control+Shift+I` (Windows) or `Command+Option+I` (macOS). Then, open Settings by clicking the gear icon at the top right of DevTools or by pressing `F1`.
```

Comment:
We know F12 is the most-used shortcut, should we consider referencing that shortcut or did we choose these others for a reason?

Reply:
Sounds good; will add F12! These were chosen only because these are the shortcuts referenced in the ["Open Edge DevTools" doc](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/open). May be worth updating that one as well.